buildSrc/src/main/java/gradlebuild/NcursesPlugin.java

```diff
@@ -54,21 +65,26 @@
         });
     }
 
-    private static String inferNCursesVersion() {
+    private static SortedSet<String> inferNCursesVersions() {
         OperatingSystem os = new DefaultNativePlatform("current").getOperatingSystem();
         if (!os.isLinux()) {
-            return "5";
+            return ImmutableSortedSet.of("5");
         }
-        for (String d : ImmutableList.of("/lib", "/lib64", "/lib/x86_64-linux-gnu", "/lib/aarch64-linux-gnu")) {
+        ImmutableSortedSet.Builder<String> builder = ImmutableSortedSet.naturalOrder();
+        for (String d : ImmutableList.of("/lib", "/lib64", "/lib/x86_64-linux-gnu", "/lib/aarch64-linux-gnu", "/usr/lib")) {
             File libDir = new File(d);
-            if (new File(libDir, "libncurses.so.6").isFile()) {
-                return "6";
+            if (new File(libDir, "libncurses.so.6").isFile() || new File(libDir, "libncursesw.so.6").isFile()) {
+                builder.add("6");
             }
             if (new File(libDir, "libncurses.so.5").isFile()) {
```

Comment:
Why not detect `libncursesw.so.5` as well?

Reply:
Because there doesn't seem to be a `w` variant for ncurses5: If you check above we link against `ncursesw` for ncurses 6 on Linux and against `ncurses` for everything else.